ppa/archive/models.py

```diff
@@ -753,6 +753,9 @@ def index_data(self):
             # hard-coded to distinguish from & sort with pages
             "item_type": "work",
             "order": "0",
+            "work_type_s": self.get_item_type_display()
+            .lower()
+            .replace(" ", "-"),  # full, excerpt, or article
```

Comment:
why the `replace()`? comment suggests there won't ever be values that have whitespace

Reply:
the actual names are "Full Work", "Excerpt", and "Article"

Reply:
I'm open to you revising this when you implement the icons if you have a different solution you prefer! I wondered if it would matter for the accessibility label you were thinking of adding — but I don't think we need to label/announce full works.

Reply:
oh, makes sense! I figured it was to make it work as a class name. can you change the comment to say "full-work" so it shows the real value that will be used?